updater/aws_test.go

```diff
@@ -270,6 +292,197 @@ func TestFilterBottlerocketInstances(t *testing.T) {
 	assert.EqualValues(t, expected, actual)
 }
 
+func TestPaginatedFilterBottlerocketInstancesAllFail(t *testing.T) {
+	instances := make([]*string, 0)
+	descOut := make([]*ecs.ContainerInstance, 0)
+	for i := 0; i < 150; i++ {
+		instanceARN := "cont-inst-br" + strconv.Itoa(i)
+		ec2ID := "ec2-id-br" + strconv.Itoa(i)
+		instances = append(instances, aws.String(ec2ID))
+		descOut = append(descOut, &ecs.ContainerInstance{
+			Attributes:           []*ecs.Attribute{{Name: aws.String("bottlerocket.variant")}},
+			ContainerInstanceArn: aws.String(instanceARN),
+			Ec2InstanceId:        aws.String(ec2ID),
+		})
+	}
+
+	responses := []struct {
+		inputLen           int
+		ContainerInstances []*ecs.ContainerInstance
+		err                error
+	}{{
+		100,
+		descOut[:100],
+		errors.New("Failed to describe container instances"),
+	}, {
+		50,
+		descOut[100:],
+		errors.New("Failed to describe container instances"),
```

Comment:
// If its an error it should not return describe output. 
```suggestion
	}{{
		100,
		nil,
		errors.New("Failed to describe container instances"),
	}, {
		50,
		nil,
		errors.New("Failed to describe container instances"),
```